firebase/ManageEvents.ts

```diff
@@ -93,22 +95,27 @@ export const getAllFutureEvents = async () => {
   }
 }
 
-export const getEventData = async(eventUid: string) => {
-  //----------------------------------------
-  // return a dummy event for the moment
-  // ---------------------------------------
-  const dummyPoint: Point = {
-    x : 40.712776,
-    y : -74.005974
-  }
-
-  return {
-    uid: eventUid,
-    title: "Title of the event right at the start",
-    location: "Central Park",
-    point: dummyPoint,
-    date: new Date(), // current date and time
-    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Dolor morbi non arcu risus quis. Mattis molestie a iaculis at. Tristique risus nec feugiat in fermentum posuere urna. Quisque sagittis purus sit amet volutpat consequat mauris nunc. Sapien eget mi proin sed libero. Condimentum mattis pellentesque id nibh tortor id aliquet lectus proin. Suspendisse faucibus interdum posuere lorem. In mollis nunc sed id semper risus. Amet consectetur adipiscing elit ut aliquam purus. Integer enim neque volutpat ac.    ",
-    imageUrl: ""
-  }
+export const getEventData = async (eventUid: string) => {
+  try {
+    const docRef = doc(db, "events", eventUid)
+    const docSnapshot = await getDoc(docRef)
+
+    if (docSnapshot.exists()) {
+      const data = docSnapshot.data() as DocumentData
+      const event: Event = {
+        uid: data.uid,
+        title: data.title,
+        location: data.location,
+        point: data.point,
+        description: data.description,
+        date: data.date.toDate(),  // Assuming 'date' is stored as a Firestore Timestamp
+        imageUrl: data.imageUrl,
+      }
+      return event
+    } else {
+      console.log("No such document!")
```

Comment:
If you want to write an error message, mention it in the comment for people to know where it comes from if they use the function at a higher level